src/OpenLoco/Math/Vector.hpp

```diff
@@ -270,4 +270,12 @@ namespace OpenLoco::Math::Vector
         };
     }
 
+    uint16_t fastSquareRoot(uint32_t distance);
```

Comment:
This looks a bit awkward. Perhaps add some extra whitespace to make it more obvious what's going on?

```suggestion
    uint16_t fastSquareRoot(uint32_t distance);

```